pom.xml

```diff
@@ -460,11 +460,11 @@
     </dependencies>
     <properties>
         <maven-javadoc-version>3.0.1</maven-javadoc-version>
-        <swagger-core-version>2.1.7-SNAPSHOT</swagger-core-version>
-        <swagger-parser-version>2.0.25-SNAPSHOT</swagger-parser-version>
+        <swagger-core-version>2.1.7</swagger-core-version>
+        <swagger-parser-version>2.0.25</swagger-parser-version>
```

Comment:
Not sure if I needed to update this

Reply:
good to have, would bump swagger-core to 2.1.8 also